code/ship/ship.cpp

```diff
@@ -18322,9 +18322,12 @@ int ship_squadron_wing_lookup(const char *wing_name)
 	}
 	else 
 	{
+		auto ch = get_pointer_to_first_hash_symbol(wing_name);
```

Comment:
A comment with explanation would be great IMO.

Reply:
k